src/async-components/structures/CompatibilityView.tsx

```diff
@@ -0,0 +1,408 @@
+/*
+Copyright 2020 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import * as React from "react";
+import { _t } from "matrix-react-sdk/src/languageHandler";
+
+// directly import the style here as this layer does not support rethemedex at this time so no matrix-react-sdk
+// scss variables will be accessible.
+import "../../../res/css/structures/ErrorView.scss";
+
+interface IProps {
+    onAccept: () => void;
```

Comment:
```suggestion
    onAccept(): void;
```

Reply:
sorry, this was meant to be in a review but I clicked 1ms too early while moving my mouse